### PR TITLE
CES-11519 Update scripts to use the hybrid idrac driver/hardware type

### PR DESCRIPTION
**Pull request by himanshusingh2**
* Please do not merge this PR for now since this PR has a dependency on other work. 
* Updated JetPack Scripts to Use "idrac" in place of "pxe_drac" and "ipmi" in place of "pxe_ipmitool".

### CODE REVIEW

File: src/pilot/discover_nodes/discover_nodes.py
Comment (rpioso) [inline src/pilot/discover_nodes/discover_nodes.py:124]: Please remove `PXE` from the constant names:

    OSPD_NODE_TEMPLATE_VALUE_PM_TYPE_IDRAC = 'idrac'
    OSPD_NODE_TEMPLATE_VALUE_PM_TYPE_IPMI = 'ipmi'

**Comment by himanshusingh2** [comment]
Please do not merge this PR for now, since this PR has a dependency on other work Richard is working on.

**Comment by rpioso** [comment]
> Hey Himanshu, nice job on your first patch! I think there is one more location where we need a change:
> https://github.com/dsp-jetpack/JetPack/blob/experimental-upstream-ironic-JS-16.1-00/src/pilot/introspect_nodes.py#L56

Great find!

**Comment by himanshusingh2** [comment]
> Looks good! Have all of these changes been tested?

**Comment by himanshusingh2** [comment]
> Looks good! Have all of these changes been tested?

Hi Chris, All of these affected Scripts have been Tested as Standalone scripts.

**Comment by rpioso** [comment]
@himanshusingh2, since this is good to go, please remove the WIP tag from this PR's subject. After that, I will merge it.

**Review by rpioso — APPROVED** [review]
Since #601 and #604 merged, this is good to go.

I am merging it.

**Pull request merged**
